frame/vamm/src/lib.rs

```diff
@@ -338,12 +357,14 @@ pub mod pallet {
 		VammIsClosed,
 		/// Tried to swap assets but the amount returned was less than the minimum expected.
 		SwappedAmountLessThanMinimumLimit,
-		/// Tried to derive invariant from base and quote asset, but the
+		/// Tried to derive invariant from [`base`](VammState::base_asset_reserves) and
+		/// [`quote`](VammState::quote_asset_reserves) asset, but the
 		/// computation was not successful.
 		FailedToDeriveInvariantFromBaseAndQuoteAsset,
-		/// Tried to perform swap operation but it would drain all base asset reserves.
+		/// Tried to perform swap operation but it would drain all [`base`](VammState::base_asset_reserves) asset reserves.
 		BaseAssetReservesWouldBeCompletelyDrained,
-		/// Tried to perform swap operation but it would drain all quote asset reserves.
+		/// Tried to perform swap operation but it would drain all
+		/// [`quote`](VammState::quote_asset_reserves) asset reserves.
 		QuoteAssetReservesWouldBeCompletelyDrained,
 	}
```

Comment:
This isn't required but you could provide links to all functions that can result in each error

Reply:
Oh, okay!